src/package.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I keep seeing this stupid standalone appearing and disappearing. We don't care about it, right @cdcarter ?